test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TypicalPersons;
+import seedu.addressbook.util.TestUtil;
```

Comment:
Nice to see that you're trying to follow the coding standard for import statements. You might have missed the minor ordering issue here.